crud/create.js

```diff
@@ -0,0 +1,45 @@
+module.exports = (binding) => {
+    let columns = binding.fields.filter(f => !f.identity).map(f => `[${f.column}]`).join(', ');
+    return `
+    CREATE PROCEDURE ${binding.spName}
+        @data ${binding.tt} READONLY
+    AS
+    SET NOCOUNT ON
+    DECLARE @result ${binding.tt}
+    BEGIN TRY
+        INSERT INTO ${binding.name} (${columns})

```

Comment:
Can we also have a generic logic for checking for already existing records and raising standard errors?

Reply:
Sure, I just wrote some generic SQL code which I didn't mean to be final ... just to test the concept.
We can change the bodies and the names of the procedures however we like. My main idea was to align with you whether the changes in [index.js](https://github.com/softwaregroup-bg/ut-port-sql/blob/feat/autoCRUD/index.js) are fine. Maybe @mlessevsg can help with the SQL.